scripts/deployment/azure/common.sh

```
@@ -0,0 +1,253 @@
+    echo "❌ $1"
+}
+
+print_step() {
+    echo "🚀 $1"
+}
+
+# 환경변수 파일 로드 함수
+load_env_file() {
+    if [[ -f ".env.production" ]]; then
+        print_info ".env.production 파일 로드 중..."
+        export $(grep -v '^#' .env.production | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)